### PR TITLE
APPS-3599 Added assertion and renamed keywords for bazaarvoice

### PR DESCRIPTION
**Pull request by miroshnikov-spryker**
## PR Description

Ticket: https://spryker.atlassian.net/browse/APPS-3599

### CODE REVIEW

File: tests/e2e/aop/bazaarvoice_pbc.robot [hunk 1]
```diff
@@ -25,25 +23,27 @@ Bazaarvoice_E2E
     Zed: click button on the PBC details page:    connect
     Zed: PBC details page should contain the following elements:    ${appPendingStatus}
     Zed: click button on the PBC details page:    configure
-    Zed: configure bazaarvoice pbc with the following data:
-    ...    || clientName      | siteId    | environment | services                                                                              | stores ||
```
Comment (AntonShopin) [inline tests/e2e/aop/bazaarvoice_pbc.robot:29]: why you have removed |stores| ? 
Does it mean that during configuration we don't check needed store checkboxes?

File: resources/steps/bazaarvoice_steps.robot [hunk 2]
```diff
@@ -60,15 +59,15 @@ Yves: post bazaarvoice review with the following data:
...
-    # END 
     Page Should Contain Element    xpath=//head//script[@id='${scriptId}']
 
-Yves: 1st product card in the catalog should contains bv inline rating
-    Page Should Contain Element    xpath=//div[contains(@class,'grid grid')]/div[1]/product-item//*[@data-bv-show='inline_rating']
\ No newline at end of file
+Yves: first product card in the catalog should contain bazaarvoice inline rating
+    Page Should Contain Element    xpath=//div[contains(@class,'grid grid')]/div[1]/product-item//*[@data-bv-show='inline_rating']
+
+Yves: bazaarvoice should send an event:
```
Comment (AntonShopin) [inline resources/steps/bazaarvoice_steps.robot:71]: We need one more keyword that should check the response of the sent request. Previously we had a bug that the request is sent, but the response was not successful (exception)

File: tests/e2e/aop/bazaarvoice_pbc.robot (hunk 1 above)
Comment (miroshnikov-spryker) [reply tests/e2e/aop/bazaarvoice_pbc.robot:29]: I think my local setup has all recent updates and there is no `stores` checkbox in this form. [](https://i.imgur.com/RWu2V0n.png)

File: resources/steps/bazaarvoice_steps.robot (hunk 2 above)
Comment (miroshnikov-spryker) [reply resources/steps/bazaarvoice_steps.robot]: Fixed keyword, added response status check

**Review by AntonShopin — APPROVED** [review]

**Pull request merged**